pages/feedback.js

```diff
@@ -48,21 +49,29 @@ const useStyles = makeStyles(theme => ({
     marginTop: theme.spacing(1),
     marginBottom: theme.spacing(1),
   },
+  pointerIcon: {
+    cursor: 'pointer',
+  },
 }))
 
 const SettingsPage = () => {
   const classes = useStyles()
+  const router = useRouter()
   const categories = ['Bug Report', 'Feedback']
-  const [sumitting, setSumitting] = useState(false)
```

Comment:
Thanks for fixing the typos 🤦

Reply:
No Problem, I'm sure you would help me out also.